src/webview/html/messageAsHtml.js

```diff
@@ -116,7 +116,8 @@ $!${divOpenHtml}
 `;
   }
 
-  const { sender_full_name, sender_email } = message;
+  const { sender_full_name } = message;
+  const sender_id = message.isOutbox ? backgroundData.ownUser.user_id : message.sender_id;
```

Comment:
Huh, is there no `sender_id` in `Outbox`?

... Huh, indeed there isn't. Well, that's our own code's fault. :stuck_out_tongue_closed_eyes: We should fix that.

This code is a good workaround for this PR's purposes, because fixing that omission is made complex by the fact we store outbox messages between runs. I'll file an issue for it.

Reply:
#3998.